cfn-resources/serverless-instance/resource-role.yaml

```diff
@@ -15,15 +15,22 @@ Resources:
             Principal:
```

Comment:
why do we still need this YAML file? wasn't execution role [here](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/examples/execution-role.yaml) created to work for all resources?

Reply:
this is autogenerated Zuhair, in all the resources, is used for submitting